tests/html/script/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -101,6 +101,16 @@ printWidth: 80
 </script><script type="invalid">
   {   "json":false  }
 </script>
+<script type="text/html">
+  <div>
+  <p>foo</p>
+  </div>
+</script>
+<script lang="html">
+  <div>
+  <p>foo</p>
+  </div>
+</script>
```

Comment:
Do not support `lang`, this can't run in browser. only type. Sorry for my mistake.

`Uncaught SyntaxError: Unexpected token '<'`

Reply:
Thank you, done :)